### DISCUSSION TITLE
Icon request: fa-car-door

### DISCUSSION THREAD

**Discussion by cristacheda** [FortAwesome/Font-Awesome#21168] [Icon Requests]
It would be really nice to have a car door icon. I'd like to use it for an open-source project for car booking.

![](https://d30y9cdsu7xlg0.cloudfront.net/png/122424-200.png)

**Comment by ShineOfFire** [comment]
up

**Comment by Sotiris-k** [comment]
+1

**Comment by harnishdesign** [comment]
+1 Here, have any possibility for car door icon.

**Comment by Los** [comment]
+1

**Comment by switchroyale** [comment]
+1

**Comment by jasonb4u** [comment]
+1, thought this would have been added by now 2.5 years after this post was made still not available, whats going on!!!

**Comment by kaloxri** [comment]
After 3 years and nothing has changed.

**Comment by tagliala** [comment]
Hi @kaloxri , upvoting existing requests is the best we can do

Please take a look at our [Icon Leaderboard](https://fontawesome.com/community/leaderboard/new), we have also a [faqs section](https://fontawesome.com/community/leaderboard/new#faqs) with a lot of information on how requests work

**Comment by jasonb4u** [comment]
To many links to other requests of MAYBE duplicates, but car door still has not been added.  How hard can it be!!!

**Comment by tikky** [comment]
+1 from me. Would like to use in GPS system as a "door status" icon

**Comment by aniskasmi** [comment]
+1

**Comment by linusanari** [comment]
I second

**Comment by hulmerous** [comment]
Still need this, in 2022

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
